pom.xml

```diff
@@ -1241,7 +1241,7 @@
             <dependency>
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
-                <version>5.1.48</version>
+                <version>8.0.23</version>
```

Comment:
We don't have to upgrade the driver to fix the timestamp precision issue along, so this is actually 2 separate changes and go by 2 separate PRs. The reason for this being MySQL/MariaDB compatibility issues (unless we already lost MariaDB compatibility). And MariaDB compatibility may be tied to AWS Aurora compatibility as well.